vercel/data_source_project.go

```diff
@@ -393,6 +393,14 @@ For more detailed information, please see the [Vercel documentation](https://ver
 				Optional:    true,
 				Computed:    true,
 			},
+			"build_machine_type": schema.StringAttribute{
+				Description: "When `on_demand_concurrent_builds` is enabled, choose the build machine: `standard` (4 vCPU, 8 GB) or `enhanced` (8 vCPU, 16GB)",
+				Optional:    true,
```

Comment:
Shouldn't be optional on the data source. Only computed. (it's a read-only property here).